cluster/terraform/config/development.tfvars.json

```diff
@@ -7,5 +7,7 @@
     "test"
   ],
   "cluster_dns_resource_group_name": "s189d01-tscdomains-rg",
-  "cluster_dns_zone": "development.teacherservices.cloud"
+  "cluster_dns_zone": "development.teacherservices.cloud",
+  "cluster_kv_cert_suffix": "development-teacherservices-cloud",
```

Comment:
“development” is in the development.sh CONFIG variable. we could pass it as a tf var so we wouldn’t need cluster_kv_cert_suffix